gazelle/generate.go

```diff
@@ -96,6 +125,47 @@ func defaultModuleName(args language.GenerateArgs) string {
 	return defaultModuleName
 }
 
+func generateRuleFromProtoPackage(protoPackageName string, protoPackage proto.Package) []*rule.Rule {
+	swiftProtoLibraryName := strings.TrimSuffix(protoPackageName, "_proto") + swiftProtoSuffix
+	swiftProtoLibrary := rule.NewRule("swift_proto_library", swiftProtoLibraryName)
+	swiftProtoLibrary.SetAttr("deps", []string{":" + protoPackageName})
+	swiftProtoLibrary.SetPrivateAttr(config.GazelleImportsKey, []string{})
+	rules := []*rule.Rule{swiftProtoLibrary}
+
+	if protoPackage.HasServices {
+		// TODO: Should we add a config to selectively generate specific flavors?
```

Comment:
If this is something that we are going to address after this PR, could you create an issue and then replace `TODO` with `GHXXX` where `XXX` is the issue number?

Reply:
Ok I created https://github.com/cgrindel/rules_swift_package_manager/issues/509 and I updated the comment to reference it.